src/main/java/lesson8Person/Person.java

```diff
@@ -0,0 +1,71 @@
+package lesson8Person;
+
+import java.time.*;
+import java.util.Objects;
+
+public class Person {
+    private final String name;
+    private final String surname;
+
+    // if I don't initialize these, I can't use the 'LocalDate.of' method, any solutions to that?
+    private int yearOfBirth = 0;
+    private Month monthOfBirth = Month.January;
+    private int dayOfBirth = 0;
+
+    private final LocalDate birthDate;
+    private double height;
+    private double weight;
+    private Period age;
```

Comment:
Wiek jako przedział? ;>

Reply:
Generalnie przechowywanie wieku jako pola nie jest dobrą praktyką, ponieważ jest to wartość, która bardzo łatwo może przestać być aktualna i trzeba ją aktualizować. Zamiast tego przetrzymuj jedynie informację o dacie urodzenia. Wiek zawsze można sobie wyliczyć.

Reply:
zrobione :)